index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Form</title>
+  </head>
+  <body>
+    <form action="https://hook.eu2.make.com/3v79iuam6rf96apg3sx7y2almjqck1qc" method="post">
+      <fieldset>
+        <legend>Personal data:</legend>
+        <div>
+          <label for="fullName">Your name</label>
+          <input type="text" id="fullName" name="fullName" required />
+        </div>
+        <div>
+          <label for="email">Your email</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="example@email.com"
+            required
+          />
+        </div>
+        <div>
+          <label for="password1">Create password</label>
+          <input type="password" id="password1" name="password1" required />
+        </div>
+        <div>
+          <label for="password2">Confirm password</label>
+          <input
+            type="password"
+            id="password2"
+            name="password2"
+            required
+          />
+        </div>
+        <div>
+          <label for="sex">Your gender</label><br />
+          <div id="sex">
+            <input type="radio" id="male" name="male" value="male" required />
+            <label for="male">Male</label><br />
+            <input
+              type="radio"
+              id="female"
+              name="female"
+              value="female"
+              required
+            />
```

Comment:
```suggestion
            <input type="radio" id="male" name="sex" value="male" required />
            <label for="male">Male</label><br />
            <input
              type="radio"
              id="female"
              name="sex"
              value="female"
              required
            />
```

`name`  маю бути спільним, якщо це саме група елементів radio або checkbox, щоб браузер розумів, що відмічати